ui/components/ui/identicon/blockieIdenticon/blockieIdenticon.component.js

```diff
@@ -19,7 +19,17 @@ const BlockieIdenticon = ({ address, diameter, alt }) => {
   return (
     <>
       <canvas ref={canvasRef} style={{ display: 'none' }} />
-      <img src={dataUrl} height={diameter} width={diameter} alt={alt || ''} />
+      <img
+        src={dataUrl}
+        height={diameter}
+        width={diameter}
+        style={
+          {
+            borderRadius: style,
+          } || null
+        }
+        alt={alt || ''}
```

Comment:
Instead of heaving this here, could we do:

```
const BlockieIdenticon = ({ address, diameter, alt = '', style }) => {
```

Reply:
done